src/doc/rustc/src/platform-support/fuchsia.md

```diff
@@ -196,6 +196,31 @@ In `config.toml`, add:
 ```toml
 [build]
 target = ["<host_platform>", "aarch64-fuchsia", "x86_64-fuchsia"]
+
+[target.<host_platform>]
+cc = "clang"
+cxx = "clang++"
+linker = "clang++"
+
+[target.x86_64-fuchsia]
+cc = "clang"
+cxx = "clang++"
+linker = "ld.lld"
+
+[target.aarch64-fuchsia]
+cc = "clang"
+cxx = "clang++"
+linker = "ld.lld"
+```
+
+Note that while `clang++` is used to link for the host platform (e.g.
+`x86_64-unknown-linux-gnu`), `lld` is used to link for Fuchsia. You may also
+want to use LLVM's versions of `ar` and `ranlib` in a similar way:
```

Comment:
What would be the reason for wanting to use these?

We do it in our toolchain build to only depend on LLVM tools but I don't think outside users will care about that.

Reply:
Removed.